tasks-by-users/Rehan-Khan-Engineer/Bikes-webpage/Rehan.css

```diff
@@ -0,0 +1,42 @@
+/* Bike ki Dukaan */
+body{ margin: 0; font-family: Arial; background: white;}
+
+#wrapper {margin: 0 auto; width: 980px; background: white;}
+
+#main-navigation{ background: #181818; padding: 10px; }
+
+#main-navigation ul { text-align: right; list-style-type: none; margin-right: 10px; }
+
+li {margin-left: 20px; display: inline-block; }
+
+#main-navigation a {color: white; text-decoration: none; text-transform: uppercase; font-size: 14px; padding: 6px;}
+
+.current { background: #8f746c; border-radius: 4px;}
+
+/* ask about this */
+#lead-banner img {width: 100%; height: 500px;}
```

Comment:
whats the question ?

Reply:
unable to understand when exactly a BOX appears on screen. is it when you give margin n border ?

Reply:
Doubt cleared @faran4engg